easybuild/tools/config.py

```diff
@@ -448,16 +450,6 @@ def get_log_filename(name, version, add_salt=False):
     return filepath
 
 
-def read_only_installdir():
-    """
-    Return whether installation dir should be fully read-only after installation.
-    """
```

Comment:
it could, but we already have the `build_option` function that serves this purpose, this placeholder function predates `build_option`